### DISCUSSION TITLE
中文Chia支持。

### DISCUSSION THREAD

**Discussion by hoffmang9** [Chia-Network/chia-blockchain#1662] [Q&A]
我为使用Google翻译表示歉意，但我想为那些会说中文的人提供一个帮助他人的地方。

**Comment by jimkoen** [comment]
我还没有用中文测试过，但[DeepL](https://www.deepl.com)，在西方语言中往往比谷歌翻译的结果更好。

**Reply by hoffmang9** [reply]
我会检查DeepL。

**Comment by Chanix** [comment]
在客户端中选择中文后，能很快的理解并使用。很多中国人（包括我）已经开始购买硬盘并参与进来。在社交软件上，也有群专门讨论chia。

或者有什么需要协助的？我能看懂英语，做简单的翻译，但写英语对我而言比较困难：）

**Reply by hoffmang9** [reply]
核实https://crowdin.com/project/chia-blockchain 的译文是否正确将是有益的。
